src/Template/Scaffolds/index.ctp

```diff
@@ -40,5 +76,34 @@
         </table>
     </div>
 
+    <?php
+    if ($hasBulkActions) {
+        $this->Form->templates([
+            'submitContainer' => '{{content}}',
+        ]);
+
+        $submitButton = $this->Form->submit(__d('crud', 'Apply'), [
+            'class' => 'btn btn-success btn-bulk-apply',
+            'div' => false,
+            'name' => '_bulk',
+        ]);
+        $this->Form->templates([
+            'inputContainer' => '<div class="form-group bulk-action-submit {{required}}">{{content}}{{help}}</div>',
```

Comment:
They need to be because one of how the templates work (embedding the button inside the template).